hypothesis-python/tests/cover/test_lookup.py

```diff
@@ -540,6 +540,18 @@ def test_resolving_recursive_type():
     assert isinstance(st.builds(Tree).example(), Tree)
 
 
+class LinkedList:
```

Comment:
Looks unused, and I'd expect that the builtin `list` type would do fine?

Reply:
(I accidentally answered this in another thread, probably below)